src/components/Checkster/feature/docs/Aboutk6Studio.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { GrafanaTheme2 } from '@grafana/data';
+import { LinkButton, Stack, Text, useStyles2, useTheme2 } from '@grafana/ui';
+import { css } from '@emotion/css';
+
+import { onDocsLinkClick } from 'components/DocsLink/DocsLink.utils';
+
+import { k6StudioLogoDarkTheme, k6StudioLogoLightTheme } from 'img';
+
+export function Aboutk6Stuido({ source }: { source: string }) {
+  const theme = useTheme2();
+  const src = theme.isDark ? k6StudioLogoDarkTheme : k6StudioLogoLightTheme;
+  const styles = useStyles2(getStyles);
+
+  return (
+    <Stack direction="column" gap={2}>
+      <Text variant="h4" element="h3">
+        <Stack direction="row" alignItems="center" gap={1}>
+          <img className={styles.logo} src={src} alt="k6 Studio logo" />
+          Get started with k6 Studio
+        </Stack>
+      </Text>
+      <Text element="p">
+        k6 Studio is a free, open source desktop application designed to help you create k6 test scripts using a visual
+        interface. Download it for free and get started with your first script in minutes.
+      </Text>
+      <div>
+        <LinkButton
+          href="https://grafana.com/docs/k6-studio/set-up/install/"
+          target="_blank"
+          onClick={() => onDocsLinkClick('https://grafana.com/docs/k6-studio/set-up/install/', source)}
```

Comment:
Since it is an external link, adding the external link icon seems fitting.

```suggestion
          target="_blank"
          icon="external-link-alt"
          onClick={() => onDocsLinkClick('https://grafana.com/docs/k6-studio/set-up/install/', source)}
```

<img width="474" height="196" alt="Image" src="https://github.com/user-attachments/assets/aed95260-b59d-46f0-8c62-9eb366204d12" />